logdna/logdna.py

```diff
@@ -39,7 +41,9 @@ def __init__(self, key, options={}):
         self.index_meta = options.get('index_meta', False)
         self.flush_limit = options.get('flush_limit', defaults['FLUSH_BYTE_LIMIT'])
         self.flush_interval = options.get('flush_interval', defaults['FLUSH_INTERVAL'])
+        self.backoff_period = options.get('backoff_period', defaults['BACKOFF_PERIOD'])
         self.tags = options.get('tags', [])
+        self.buf_size_limit = options.get('buf_retention_limit', defaults['BUFFER_BYTE_LIMIT'])
```

Comment:
We're using 3 different names here - `buf_size_limit`, `buf_retention_limit`, and `BUFFER_BYTE_LIMIT`. Can we choose 1 and be consistent?